src/org/sosy_lab/java_smt/solvers/z3/Z3SolverException.java

```diff
@@ -1,21 +0,0 @@
-// This file is part of JavaSMT,
-// an API wrapper for a collection of SMT solvers:
-// https://github.com/sosy-lab/java-smt
-//
-// SPDX-FileCopyrightText: 2020 Dirk Beyer <https://www.sosy-lab.org>
-//
-// SPDX-License-Identifier: Apache-2.0
-
-package org.sosy_lab.java_smt.solvers.z3;
-
-import org.sosy_lab.java_smt.api.SolverException;
-
-/** This exception is thrown by native Z3 code, through our customized JNI bindings. */
```

Comment:
It seems the bindings need to be fixed as well.

Reply:
There is no need to update anything. Since several years, JavaSMT relies on the original JNI bindings directly from Z3. This exception was never thrown from there. This class was taken over from CPAchecker in 2015, and it was never really used.